src/components/tx/ApprovalEditor/ApprovalValueField.tsx

```diff
@@ -66,24 +66,27 @@ export const ApprovalValueField = ({ name, tx }: { name: string; tx: ApprovalInf
             label={label}
             name={name}
             fullWidth
+            helperText={helperText}
+            onFocus={(field) => {
+              if (!readOnly) {
+                field.target.select()
+              }
+            }}
+            margin="dense"
+            variant="standard"
             error={!!fieldState.error}
             size="small"
             onBlur={onBlur}
             inputRef={ref}
             InputProps={{
               ...params.InputProps,
               sx: {
-                paddingTop: '4px',
-                paddingBottom: '4px',
-                paddingLeft: 1,
                 flexWrap: 'nowrap !important',
+                '&::before': {
+                  border: 'none !important',
+                },
+                border: 'none !important',
               },
-              startAdornment: (
-                <Box display="flex" flexDirection="row" alignItems="center" gap="4px">
-                  <TokenIcon size={32} logoUri={tx.tokenInfo?.logoUri} tokenSymbol={tx.tokenInfo?.symbol} />
-                  <Typography>{tx.tokenInfo?.symbol || shortenAddress(tx.tokenAddress)}</Typography>
-                </Box>
-              ),
             }}
             inputProps={{
               ...params.inputProps,
```

Comment:
I can't reach it here since its not part of the diff but I would add this to the `InputLabelProps` so that the color doesn't change when its in `readOnly` mode otherwise it seems like the field can be edited even though it is disabled:

```
sx: {
    color: (theme) => (readOnly ? `${theme.palette.text.secondary} !important` : undefined),
},
```